### ISSUE TITLE
Create admin user with default password

### ISSUE THREAD

**Issue by lcabrini** [lcabrini/sokms#22]
Now that registration is disabled, I think we need to automatically create an admin user with a default password. Otherwise it is impossible to log in with a new database.

**Comment by CollinsLord** [comment]
Okay

**Comment by lcabrini** [comment]
@CollinsLord do you want to try to do this?

**Comment by CollinsLord** [comment]
Lol yes please i will try

**Comment by Prince-linux** [comment]
+1 @CollinsLord

**Comment by CollinsLord** [comment]
Hello i have done something about this issue but am not too sure.

**Comment by lcabrini** [comment]
Yes, I asked a question on the pull request. Can you please check and confirm?

**Comment by lcabrini** [comment]
Wait, pull request message says "Added dashboard and some controls". But this issue is about creating an admin user. Now I am really confused. What changes did I actually pull in with pull request #24 ?

Is there now an admin user with a default password? If so, what is the username/password?

**Comment by CollinsLord** [comment]
hahahah @lcabrini  you made my evening this pull request has made me laugh myself.
I was able to create the admin page ,for what i used on my system is admin@email.com and the password is 123456

**Comment by CollinsLord** [comment]
I can't stop laughing @lcabrini i love your comment.

**Comment by lcabrini** [comment]
(1/1)                                         ErrorException                                                                          Undefined property: Illuminate\Routing\Redirector::$back
--
in LoginController.php (line 27)
at HandleExceptions->handleError(8,  'Undefined property: Illuminate\\Routing\\Redirector::$back',  '/home/lorenzo/Git/sokms/sokms/app/Http/Controllers/LoginController.php',  27, array('request' => object(Request)))in LoginController.php (line 27)
at LoginController->login(object(Request))
at call_user_func_array(array(object(LoginController), 'Login'), array(object(Request)))in Controller.php (line 55)
at Controller->callAction('Login', array(object(Request)))in ControllerDispatcher.php (line 44)
at ControllerDispatcher->dispatch(object(Route), object(LoginController), 'Login')in Route.php (line 203)
at Route->runController()in Route.php (line 160)
at Route->run()in Router.php (line 574)
at Router->Illuminate\Routing\{closure}(object(Request))in Pipeline.php (line 30)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in SubstituteBindings.php (line 41)
at SubstituteBindings->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in VerifyCsrfToken.php (line 65)
at VerifyCsrfToken->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in ShareErrorsFromSession.php (line 49)
at ShareErrorsFromSession->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in StartSession.php (line 64)
at StartSession->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in AddQueuedCookiesToResponse.php (line 37)
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in EncryptCookies.php (line 59)
at EncryptCookies->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Pipeline.php (line 102)
at Pipeline->then(object(Closure))in Router.php (line 576)
at Router->runRouteWithinStack(object(Route), object(Request))in Router.php (line 535)
at Router->dispatchToRoute(object(Request))in Router.php (line 513)
at Router->dispatch(object(Request))in Kernel.php (line 176)
at Kernel->Illuminate\Foundation\Http\{closure}(object(Request))in Pipeline.php (line 30)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php (line 30)
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in TransformsRequest.php (line 30)
at TransformsRequest->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in ValidatePostSize.php (line 27)
at ValidatePostSize->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in CheckForMaintenanceMode.php (line 46)
at CheckForMaintenanceMode->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))in Pipeline.php (line 53)
at Pipeline->Illuminate\Routing\{closure}(object(Request))in Pipeline.php (line 102)
at Pipeline->then(object(Closure))in Kernel.php (line 151)
at Kernel->sendRequestThroughRouter(object(Request))in Kernel.php (line 116)
at Kernel->handle(object(Request))in index.php (line 53)
at require_once('/home/lorenzo/Git/sokms/sokms/public/index.php')in server.php (line 21)

**Comment by lcabrini** [comment]
That was upon trying to login with the given credentials.

**Comment by CollinsLord** [comment]
Alright then you have to add new user and use the tinker to set the user to admin 
commans
php artisan tinker
 $user=App\User::all();
$user=App\User::find(2);
$user->admin=1
$user->save();

**Comment by CollinsLord** [comment]
That was the whole logic i couldn't set the the default password and user name. Just home am not stressing you too much.

**Comment by lcabrini** [comment]
OK, I get it. But the whole point of this particular issue was to in code automatically create the admin user with a default password so that, when you first install and do the migrations, the admin user is created and you can log in.

The reason is that we disabled registration, so an inital user must exist.
